idd/Energy+.idd.in

```diff
@@ -56705,18 +56739,51 @@ Coil:Heating:WaterToAirHeatPump:EquationFit,
         \units m3/s
         \ip-units gal/min
         \autosizable
-   N3,  \field Gross Rated Heating Capacity
+   N3,  \field Gross Reference Heating Capacity
         \note Heating capacity not accounting for the effect of supply air fan heat
         \required-field
         \type real
         \minimum> 0.0
         \units W
         \autosizable
-   N4,  \field Gross Rated Heating COP
+   N4,  \field Gross Reference Heating COP
+        \note Gross heating COP at reference conditions, when the capacity is autosized, or
+        \note rated conditions.
```

Comment:
It's just Gross heating COP at reference conditions whether hard-sized or autosized, isn't it?